src/common/hooks/campaigns.ts

```diff
@@ -4,12 +4,21 @@ import { useKeycloak } from '@react-keycloak/ssr'
 
 import { endpoints } from 'service/apiEndpoints'
 import { authQueryFnFactory } from 'service/restRequests'
-import { CampaignDonation, CampaignResponse, CampaignType } from 'gql/campaigns'
+import {
+  CampaignDonation,
+  CampaignResponse,
+  CampaignType,
+  AdminCampaignResponse,
+} from 'gql/campaigns'
 
 export function useCampaignList() {
   return useQuery<CampaignResponse[]>(endpoints.campaign.listCampaigns.url)
 }
 
+export function useCampaignAdminList() {
+  return useQuery<AdminCampaignResponse[]>(endpoints.campaign.listAdminCampaigns.url)
+}
+
```

Comment:
We may need to send authorization JWT in the headers of this request, otherwise, the backend will throw 401 unauthorized error.

```suggestion
export function useCampaignAdminList() {
  const { keycloak } = useKeycloak<KeycloakInstance>()
  return useQuery<AdminCampaignResponse[]>(
    endpoints.campaign.listAdminCampaigns.url,
    authQueryFnFactory<CampaignResponse>(keycloak?.token),
  )
}

```

Reply:
Added, route is not public so it's needed.